### ISSUE TITLE
Note of obsoletion: PHENOTYPE IRIs

### ISSUE THREAD

**Issue by matentzn** [monarch-initiative/dipper#909]
The following IRIs will not be supported by uPheno anymore:

https://github.com/obophenotype/upheno-dev/blob/master/src/curation/qc/required_allele_phenotype.txt
https://github.com/obophenotype/upheno-dev/blob/master/src/curation/qc/required_other_classes.txt

If anyone cares about them, they should let me know.

**Comment by TomConlin** [comment]
Does this leave the thousands of remaining PHENOTYPE IRIs not on this list supported?

**Comment by matentzn** [comment]
Yes, but properly. I will work with flybase to get them to support a proper phenotype module for their anatomy ontology, which gets rid of the FBBT*PHENOTYPE terms. I will also support all GO phenotype terms directly from uPheno.

**Comment by matentzn** [comment]
The goal is to not have a single IRI ending with PHENOTYPE
